back-end/api/server.js

```diff
@@ -101,7 +103,10 @@ server.get('/google/redirect', passport.authenticate('google'), (req, res) => {
     console.log('REDIRECT SUCCESS-PASSPORTREQ:', req._passport.session.user);
     
     
-            res.cookie('userID', req._passport.session.user.oauth_id);  
+    console.log('session:', req._passport.session.user.oauth_id)
+    req.session.token = 'EXPRESS' //testing if a token can be accessed on the front end
```

Comment:
quote "Having the token in session we can set the correct cookie that can later be used on the client side"...thanks for google